src/backend/distributed/commands/alter_table.c

```diff
@@ -210,6 +212,8 @@ static char * GetAccessMethodForMatViewIfExists(Oid viewOid);
 static bool WillRecreateForeignKeyToReferenceTable(Oid relationId,
 												   CascadeToColocatedOption cascadeOption);
 static void WarningsForDroppingForeignKeysWithDistributedTables(Oid relationId);
+static void ErrorForUnsupportedCascadeObjects(Oid relationId);
```

Comment:
```suggestion
static void ErrorIfUnsupportedCascadeObjects(Oid relationId);
```

This reads more natural to me. What do you think?

Reply:
A quick search shows that most (all but one) follow the `ErrorIf....` pattern. The one that doesn't also uses an `ErrorOutFor....` pattern.

I think it would be nice to stick to the precedence of `ErrorIf...` pattern for a more consistent code base.